browser/src/control/Control.ContextMenu.js

```diff
@@ -179,7 +179,7 @@ L.Control.ContextMenu = L.Control.extend({
 		} else {
 			L.installContextMenu({
 				selector: '.leaflet-layer',
-				className: 'cool-font',
+				className: 'cool-font on-the-fly-context-menu',
```

Comment:
Nit: maybe "dynamic-context-menu" or "automatic-context-menu"? But sure, naming is hard. :-)